Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM flyimg/base-image:1.1.0
 
+# Install other file processors.
+RUN apt-get update
+RUN apt-get install -y ghostscript ffmpeg
```

Comment:
Dependencies are added in the flying/base-image.

Although I would be happy to move them all here. 

But @sferjani probably has a good reason to keep them there.

Let's just put them in one place.

Reply:
This one's up to you guys. It might be beneficial to leave these in this file so if someone didn't want to use those features they could comment them out.

Reply:
Yes, I'd agree to keep it here.